packages/react/src/components/SuggestionMenu/hooks/useSuggestionMenuKeyboardNavigation.ts

```diff
@@ -7,7 +7,8 @@ export function useSuggestionMenuKeyboardNavigation<Item>(
   editor: BlockNoteEditor<any, any, any>,
   query: string,
   items: Item[],
-  onItemClick?: (item: Item) => void
+  isEmojiMenu: any,
```

Comment:
Great job with the keyboard functionality. I'd like to propose to make cleanly separate this hook of "emoji" related concerns. We could do this by changing `isEmojiMenu` to a property `gridWidth`, and set that to 1 for the suggestionmenu or 10 for the emoji menu